Dockerfile

```diff
@@ -0,0 +1,195 @@
+#     need those files to persist).
+#
+# Build arguments:
+#  - TASK_RUNNER_IMAGE (required): name of the task runner Docker image (see description below).
+#  - EXTRA_WP_PLUGINS_DIR (optional): directory, in the build context, containing zip files of extra WordPress plugins
+#      to extract in the assets/plugins/ directory. Can be used, for example, to add plugins not available in the
+#      WordPress' plugin directory.
+#  - EXTRA_VENDOR_ASSETS_DIR (optional): directory, in the build context, containing zip files to extract in the
+#      assets/vendor/ directory.
+#  - APACHE_USER_ID (optional, default=10000): user id of the created Apache user (www-data). Setting the id gives more
+#      control over permissions if you mount a directory that must be writable by the website
+#      (ex: the assets/upload/ directory). For security reasons, it's recommended that the id be >= 10000, unless you
+#      know what you do.
+#  - APACHE_GROUP_ID (optional, default=10001): group id for the created Apache user's group (www-data). For security
+#      reasons, it's recommended that the id be >= 10000, unless you know what you do.
+#
+# Special runtime environment variables (used only when running the container):
+#  - WP_*: variables prefixed with WP_ are used to set WordPress constants. For example `WP_DB_NAME=test` will set the
+#      `DB_NAME` WordPress constant to "test". See dockerfiles/www/configs/wordpress/wp-config.php for details.
+#  - APP_NO_HTTPS: if "true", informs various components in the container to allow non-HTTPS requests. If not set, or
+#      not set to "true", those components will act in "required HTTPS" mode.
+#
+# The build process is multi-stage, and the "target" you want to build the website is "production".
+#
+# The build process is two-step:
+#   1) Some static assets (ex: CSS files) are built from source files (ex: SCSS). This process is done by a
+#      "task runner" image that generates those files locally. You must supply the name of the task runner Docker image
+#      with the `TASK_RUNNER_IMAGE` build argument. See the `dockerfiles/runner/Dockerfile` file to build the
+#      task runner image.
+#   2) The production web server is built. It contains PHP-FPM, Apache, WordPress, the required WordPress plugins, and
+#      all the site's files, including the files generated by the task runner and the custom themes.
+#
+# Example build:
+#
+#   `docker build --target production --build-arg TASK_RUNNER_IMAGE=task-runner:v2`
+
+ARG TASK_RUNNER_IMAGE
+
+###
+# Task runner stage.
+#
+# The task runner compiles some site's assets from their source files. It outputs the generated files in a local dist/
+# directory.
+###
+FROM ${TASK_RUNNER_IMAGE} as task-runner
+
+WORKDIR /home/node/app
+
+COPY --chown=node framework src/framework
+COPY --chown=node fw-child src/fw-child
+
+RUN compile-sass.sh src dist
+
+CMD ["echo", "Done"]
```

Comment:
Do you want to be more verbose on the steps if you are echoing "Done" here:
- Which step is it ?
- Should you echo on other steps further down the docker build ?

Reply:
This is only the command that would be run if calling `docker run  ...`
Since it is a multi-stage build with a `CMD` applied later on, it essentially does nothing here.
The docker build logs should be sufficient (and more verbose actually) to see that the previous `RUN` command completed.